src/App.js

```diff
@@ -1,22 +1,25 @@
-import {
-  BrowserRouter as Router, Routes, Route,
-} from 'react-router-dom';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
```

Comment:
You successfully created a single page application (SPA) using react-router-dom to serve each page without reloading 🙌🔥